packages/mobile/package.json

```diff
@@ -89,7 +89,7 @@
     "@redux-devtools/remote": "0.8.0",
     "@reduxjs/toolkit": "1.6.1",
     "@sayem314/react-native-keep-awake": "1.2.2",
-    "@sentry/react-native": "5.33.1",
+    "@sentry/react-native": "6.4.0",
```

Comment:
required an sdk version bump to get some bugfixes to enable navigation interaction metrics

Reply:
Are there any differences we should be aware of?

Reply:
most of the updates were on the side of performance monitoring which we didn't have until this PR so should be good: https://docs.sentry.io/platforms/react-native/migration/v5-to-v6/

we should keep an eye out though